### ISSUE TITLE
The custom dataset always training result closet 0.

### ISSUE THREAD

**Issue by LEE-SEON-WOO** [facebookresearch/detr#84]
Hello

I want to reproduce your wonderful research with my own dataset.

I trained a total of 1594 objects based on about 1200 images for 15 objects including the background.

However, even when learning, if you test in the middle, the result of the training will always be 0.

What I changed was that I changed my data from voc to coco and num_queries from 100 to 30.

![image](https://user-images.githubusercontent.com/12380660/84587125-8e3d6f00-ae57-11ea-9481-05d0a8de0303.png)

However, as a result of analyzing the log.txt file, I can see that loss and error are smaller. Can I know what I am not doing well?

![image](https://user-images.githubusercontent.com/12380660/84587133-a01f1200-ae57-11ea-9894-050a1dbf6201.png)

**Comment by fmassa** [comment]
Hi,

Thanks for you interest in using DETR.

Given the size of your dataset, it might be preferable to start from a pre-trained model on COCO, and finetune only the last layer, while keeping the 100 object queries as default.

I would recommend following the discussion in https://github.com/facebookresearch/detr/issues/9, there are a lot of useful information there already.

I propose to move the discussion there so that it is localized in a single place.
